console/src/main.tsx

```diff
@@ -79,6 +79,11 @@ const MainUnderContext = (): ReactElement => {
   const theme = Layout.useThemeProvider();
   const cluster = Cluster.useSelect();
   const activeRange = Range.useSelect();
+  useEffect(() => {
```

Comment:
We should extract this into a named hook or add a comment explaining why we put it there.